.ebextensions/env-file-creation.config

```diff
@@ -0,0 +1,41 @@
+# Creates an .env file from AWS SSM Parameter Store
+
+commands:
+  01-create-env:
+    command: "/tmp/create-env.sh"
+
+files:
+  "/tmp/create-env.sh":
+      mode: "000755"
+      content : |
+        #!/bin/bash
+        # Reach into the undocumented container config
```

Comment:
what do you mean by "undocumented container config"?

Reply:
See the commit message; tldr - AWS did not document reading directly from this particular file

Reply:
wow this is even more sus than I initially thought

Reply:
Actually wait, not undocumented -
https://aws.amazon.com/premiumsupport/knowledge-center/view-environment-properties-elastic-beanstalk/

Reply:
@LoneRifle could we discuss with the team at one of Form's sync-ups? i think we need greater clarity around these files so that we can assess whether it is wise to rely on them in production